build.xml

```diff
@@ -773,16 +773,45 @@ more information about each.
             </fileset>
         </chmod>
         <mkdir dir="${build.dir}/run"/>
+        <if>
+            <and>
+                <islessthan arg1="${java.specification.version}" arg2="11"/>
+                <isset property="opendcs.jfr"/>
+            </and>
+            <then>
+                <property name="jfrOpts" value=""/>
+                <echo level="warning">Can only run Java Flight recorder on JDK 11 or above</echo>
+            </then>
+            <elseif>
+                <isset property="opendcs.jfr"/>
+                <then>
+                    <property name="can_jfr" value="true"/>
+                    <property name="jfrOpts"
+                                value="-XX:StartFlightRecording=disk=true,dumponexit=true,filename=${build.dir}/run/${opendcs.app}.recording.jfr,settings=profile"/>
```

Comment:
I'm also going to change the file name here to include the profile. Sometimes I'm testing several at once.